File: src/app/src/main/java/servicestack/net/kotlinlinqexamples/support/Customer.java
package servicestack.net.kotlinlinqexamples.support;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by mythz on 7/26/2015.
 */
public class Customer {
    public String customerId;
    public String companyName;
    public String address;
    public String city;
    public String region;
    public String postalCode;
    public String country;
    public String phone;
    public String fax;
    public List<Order> orders;

    public Customer() {
        orders = new ArrayList<Order>();
    }

    @Override
    public String toString() {
        return "Customer{" +
                "customerId='" + customerId + '\'' +
                ", companyName='" + companyName + '\'' +
                ", orders='" + orders.size() + '\'' +
                '}';
    }
}

File: app/src/main/java/servicestack/net/kotlinlinqexamples/support/Customer.kt
package servicestack.net.kotlinlinqexamples.support

import java.util.ArrayList

class Customer {
    var customerId: String? = null
    var companyName: String = ""
    var address: String? = null
    var city: String? = null
    var region: String? = null
    var postalCode: String? = null
    var country: String? = null
    var phone: String? = null
    var fax: String? = null
    var orders: List<Order>

    init {
        orders = ArrayList<Order>()
    }

    override fun toString(): String {
        return "Customer{" + "customerId='" + customerId + '\'' + ", companyName='" + companyName + '\'' + ", orders='" + orders.size + '\'' + '}'
    }
}
